nosnakecase.go

```diff
@@ -52,7 +52,7 @@ func report(pass *analysis.Pass, pos token.Pos, name string) {
 
 	// If prefix is Test or Benchmark, Fuzz, skip
 	// FYI https://go.dev/blog/examples
-	if strings.Contains(name, "Test") || strings.Contains(name, "Benchmark") || strings.Contains(name, "Fuzz") {
+	if strings.HasPrefix(name, "Test") || strings.HasPrefix(name, "Benchmark") || strings.HasPrefix(name, "Fuzz") {
```

Comment:
It's still not perfect because you will apply this matching on every Ident and not only functions inside test files with the right signature.
But it's much better, this will reduce false-positive and false-negative. :+1:

Reply:
Hi, @ldez 
I know. So I've planned to improve this linter 👍